src/state/api/utils.ts

```diff
@@ -64,7 +76,8 @@ export const generateTidligstMuligHeltUttakRequestBody = (args: {
   } = args
 
   return {
-    simuleringstype: getAfpSimuleringstypeFromRadio(afp),
```

Comment:
Spørre teamet: vi har tidligere sendt afp parameteren med i TMU-request for brukere med UT, mens den nullstilles i request til alderspensjon. Var det en feil, eller var det med vilje?

Reply:
Avklart: det er fordi TMU ikke kalles for brukere med uføregrad